portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,125 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body{
+  margin:auto;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.landing-section{
+  background: paleturquoise;
+  height: 100vh;
+  width: 100vw;
+  flex-direction: column;
+  align-items: center;
+  display: flex;
+}
+
+#title{
+  position: absolute;
+  margin-top: 41vh;
+  font-size: 56pt;
+  margin-left: 0.75vw;
+  font-family: 'Montserrat', serif;
+}
+
+#tri{
+  width: 35vw;
+}
+
+.social-links{
+  display: flex;
+  justify-content: space-evenly;
+  width: 20vw;
+}
+
+a > img{
+  width: 2vw;
+}
+
+#biography{
+  padding-top: 5vh;
+  width: 50vw;
+  font-size: 24pt;
+  font-family: "Open Sans", serif;
+}
+
+.transition-bar{
+  width: 100vw;
+  height: 10vh;
+  background-color: black;
+}
+
+#project-page{
+  background-color: aliceblue;
+  display: flex;
+  width: 100vw;
+  height: 300vh;
+  flex-direction: column;
+  align-items: center;
+}
+
+.subtitle{
+  padding-top: 6vh;
+  padding-bottom: 6vh;
```

Comment:
Just a quick note on margins and padding - based on context, I'd guess you actually want margin here! Margins also have stackable properties, which is nice when styling your page.

Feel free to give this a quick read: https://uxengineer.com/padding-vs-margin/

The first half of the article has some nice visuals to give you a better idea of what I mean :)

Reply:
That's a useful read. Padding is best here because the subtitle is a sibling of the rest of the project section elements, based on how I set up the html.